fragments/rgw/tier0_pacific_rgw/01_setup_prerequisites.yaml

```diff
@@ -0,0 +1 @@
+../../setup/test_setup_configure_system.yaml
```

Comment:
This being a suite, it should move to `suites/pacific/rgw/tier-0_test-build-sanity`

Reply:
Sure, I will do that.